template/content/test/child-process.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var console = require('console');
+var path = require('path');
+var spawn = require('child_process').spawn;
+var setTimeout = require('timers').setTimeout;
+
+var TestCluster = require('./lib/test-cluster.js');
+
+var serverFile = path.join(
+    __dirname, '..', 'server.js'
+);
+
+TestCluster.test('spin up server', {
+    appCount: 0
+}, function t(cluster, assert) {
+    var args = [
+        serverFile,
+        '--port', '0'
+    ];
+
+    var hostPortList = cluster.hyperbahnCluster.hostPortList;
```

Comment:
This seems like it would be easier to import your server as a module and call it rather than spawning a new process. Wouldn't you want a fatal exception when running the tests to fail fast and exit?